### PR TITLE
Added scripts to generate output conveniently

### PR DESCRIPTION
**Pull request by gapra-msft**
Covers git clone and upload download cases so far. Next PR to include fio

### CODE REVIEW

**Review by vibhansa-msft — APPROVED** [review]

**Comment by vibhansa-msft** [comment]
Our ResNet50 pipeline shall be taken as a perftest pipeline and may be these kinds of tests we can add there so that on weekly basis from our main branch this perftest and comparison against v1 is done. Periodically or on any major change we can just take the results from here and see if anything has gone up or down.

File: test/scripts/pwrite.sh [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+thread=$1
+count=$2
+size=$3 
+mntPath=$4 
+outputPath=$5 
+sed_line=$6
 
 start_time=`date +%s`
-time seq 1 $2 | parallel --will-cite -j $1 -I{} dd if=/dev/zero of=$4/$3_{}.tst bs=1M count=$3
+time seq 1 $count | parallel --will-cite -j $thread -I{} dd if=/dev/zero of=$mntPath/$size_{}.tst bs=1M count=$size
```
Comment (vibhansa-msft) [inline test/scripts/pwrite.sh:11]: Here replace /dev/zero with /dev/random. Write now this test case is generating a file which has only 0, which will be very fast in terms of generation and disk writing and does not qualify to be a real-life test case. /dev/random will generate random data.

File: test/scripts/git_clone.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# To run this script from your workspace execute this command
+#   ./test/scripts/run.sh /mnt/blob_mnt /mnt/blobfusetmp ./config.yaml ./v1.cfg 2&> results.txt
```
Comment (vibhansa-msft) [inline test/scripts/git_clone.sh:4]: we need to correct these steps here as this might not require these many inputs for git clone.

Comment (vibhansa-msft) [inline test/scripts/git_clone.sh:4]: "run.sh" commands in comments is not correct

Comment (gapra-msft) [reply test/scripts/git_clone.sh:4]: fixed, will push in a bit

Comment (gapra-msft) [reply test/scripts/git_clone.sh]: it does need the same inputs

**Review by souravgupta-msft — APPROVED** [review]

**Pull request merged**